readme.md

```diff
@@ -115,6 +115,9 @@ ___
 * [rennsax-2021](https://github.com/rennsax/ICS-lab)
 * [ghazariann-2021](https://github.com/ghazariann/SJTU-computer-system-fundamentals)
 
+> SE2301 & SE2302 计算机系统基础
```

Comment:
@kiwi142857 Could you move the comment to the head of this chapter? Like:

> 2014 级及以前：SE105 & SE106 - 程序设计
> 2015 级及以后：SE117 - 程序设计与数据结构
> 工科平台：SE232 - 程序设计与数据结构

Reply:
sorry for that, I have reformatted it.